src/Hipace.cpp

```diff
@@ -188,17 +188,16 @@ Hipace::DefineSliceGDB (const amrex::BoxArray& ba, const amrex::DistributionMapp
 
     // Slice Geometry
     constexpr int lev = 0;
-    const int dir = AMREX_SPACEDIM-1;
     // Set the lo and hi of domain and probdomain in the z direction
     amrex::RealBox tmp_probdom = Geom(lev).ProbDomain();
     amrex::Box tmp_dom = Geom(lev).Domain();
-    const amrex::Real dx = Geom(lev).CellSize(dir);
-    const amrex::Real hi = Geom(lev).ProbHi(dir);
+    const amrex::Real dx = Geom(lev).CellSize(Direction::z);
+    const amrex::Real hi = Geom(lev).ProbHi(Direction::z);
     const amrex::Real lo = hi - dx;
```

Comment:
```suggestion
    const amrex::Real dz = Geom(lev).CellSize(Direction::z);
    const amrex::Real hi = Geom(lev).ProbHi(Direction::z);
    const amrex::Real lo = hi - dz;
```

Reply:
Ah, good catch! Thanks for fixing that, too